### PR TITLE
[WIP] - EN/00-01.html Dropdown Menu Select Option to Change Languages

### PR DESCRIPTION
**Pull request by DaricusDuncan**
**[Work In Progress]** for **EN/00-01.html** [Web Page]

Implementation of drop-down menu for the English 00-01.html page.

Please critique and help make better.

Link to live web page is located here: https://daricusduncan.github.io/regex-tutorial/

### CODE REVIEW

**Comment by niccokunzmann** [comment]
Hi, this looks nice. If you like to put this into the footer: https://github.com/CoderDojoPotsdam/regex-tutorial/blob/master/_layouts/tutorial.html#L118
then, it can be seen on all pages.

**Comment by niccokunzmann** [comment]
Github Pages use Jekyll to generate the web page.
I use this for a reference: https://learn.cloudcannon.com/jekyll-cheat-sheet/ but it does only give an overview and no introduction to Jekyll.

If you set up the Jekyll software locally, you get feedback in under a second if you change something. Github pages are slow and I never know if they show the updated version.

You can have a look what modifying this file and trying out different Jekyll commands does here:
https://github.com/CoderDojoPotsdam/regex-tutorial/blob/master/_layouts/tutorial.html
Maybe there is a Jekyll tutorial.

An other idea, if you like to put this into the footer of every page:
In the page, we can exchange the language part and use `{{ page.url }}` or something like it to build together the url for all pages.

    ../en/00-01.html

**Comment by DaricusDuncan** [comment]
@niccokunzmann Thank you so much for the advice and materials. 

I am making some changes now and will get Jekyll set up locally so that I can implement pages.url or just learn Jekyll and leverage it. 

I will encapsulate all of my code so far into a footer like suggested.

I appreciate the Jekyll cheat sheet and will be studying over that now.

File: en/00-01.html [hunk 1]
```diff
@@ -17,3 +17,29 @@
...
+
+<script>
+function changeLanguage() {
+ var getLanguageMenu = document.getElementById('languageMenu');
+ var getLanguageHref = getLanguageMenu.options[getLanguageMenu.selectedIndex].getAttribute('href');
+ window.location.assign(getLanguageHref);
+ 
+}
+</script>
+<script src="http://code.jquery.com/jquery-3.2.1.min.js"></script>
```
Comment (niccokunzmann) [inline en/00-01.html]: This site shall also work off-line, without any Internet connection.
This is not important now. Before I merge, I would like to make sure that a downloaded version does not load resources from other locations. This is also a privacy concern as I do not want this page to tell jquery.com that the viewer is interested in this site.

Comment (DaricusDuncan) [reply en/00-01.html]: @niccokunzmann I can try to figure out a solution to my problem without using JQuery or using the downloaded version of JQuery.

I am still working on this issue and will be continuing the setup of Jekyll locally.

I am one trumpet out of the band here so if there is anything you just do not like, let me know and I can change it.

**Comment by niccokunzmann** [comment]
All is fine so far. What does "one trumpet out of the band" mean?

**Comment by DaricusDuncan** [comment]
@niccokunzmann It is a figure of speech saying that I am one person helping out the group. The person playing the trumpet has to be in tune with the rest of the band otherwise the band is off.

I am just trying to help the team to the best of my abilities, learn from everyone, and be a team player for the project.

**Comment by DaricusDuncan** [comment]
@niccokunzmann I am still learning how to add my code to the footer so that I can use {{page.url}}. I am reading and watching videos to help. Just wanted to update you.

**Comment by niccokunzmann** [comment]
I just checked it out .. cool! This is what I hoped for.
I will merge it since the language feature works online.
It does not work offline but that will be a new issue.

**Comment by niccokunzmann** [comment]
@DaricusDuncan, I invite you to collaborate on this: https://github.com/CoderDojoPotsdam/regex-tutorial/invitations I will be able to merge other people's pull-requests if you accept.

**Pull request merged**